src/hosted-buttons/index.js

```diff
@@ -34,24 +42,58 @@ export const getHostedButtonsComponent = (): HostedButtonsComponent => {
         selector,
       });
 
-      // $FlowFixMe
-      Buttons({
+      const createOrder = buildHostedButtonCreateOrder({
+        enableDPoP,
         hostedButtonId,
-        style,
-        onInit,
+        merchantId,
+      });
+      const onApprove = buildHostedButtonOnApprove({
+        enableDPoP,
+        hostedButtonId,
+        merchantId,
+      });
+
+      const buttonOptions = {
+        createOrder,
+        hostedButtonId,
+        merchantId,
+        onApprove,
         onClick,
-        createOrder: buildHostedButtonCreateOrder({
-          enableDPoP,
-          hostedButtonId,
-          merchantId,
-        }),
-        onApprove: buildHostedButtonOnApprove({
-          enableDPoP,
-          hostedButtonId,
-          merchantId,
-        }),
-      }).render(selector);
+        onInit,
+        style,
+      };
+
+      if (shouldRenderSDKButtons(fundingSources)) {
+        const { flexDirection } = getFlexDirection({ ...style });
+
+        appendButtonContainer({ flexDirection, selector });
+
+        // Only render 2 buttons max
+        fundingSources.slice(0, 2).forEach((fundingSource, index) => {
+          // $FlowFixMe
+          const standaloneButton = Buttons({
+            ...buttonOptions,
+            style: {
+              ...style,
+              color: getButtonColor(style.color, fundingSource),
+            },
+          });
+
+          if (standaloneButton.isEligible()) {
+            standaloneButton.render(
+              index === 0 ? "#ncp-primary-button" : "#ncp-secondary-button"
+            );
+          } else {
+            getLogger().error(`${fundingSource} is not eligible.`);
+          }
+        });
+      } else {
```

Comment:
I may be misunderstanding this logic, but should we check the eligible buttons before we slice the first two buttons?  

For example, if they pass `fundingSources: ["paypal", "venmo"]`, and Venmo is not eligible for a buyer, we would want to show PayPal and PayLater (like we do for the smart stack).  

I was thinking we'd use [`determineEligibleFunding()`](https://github.com/paypal/paypal-checkout-components/blob/main/src/funding/funding.js#L186) like we do for the smart stack but if this logic falls back correctly, I'm good with it 😄

Reply:
All of this logic will need to be refactored when we get the updated response from NCP team.

They won't be passing us the `fundingSources` array so we will need entirely new logic on how to handle rendering the second button.

I think we will be better able to update this logic once we have the updated response from NCP team

Reply:
Ahh, sounds good 👍  Thanks!